cmd/lncli/commands.go

```diff
@@ -881,6 +885,40 @@ func sendPayment(ctx *cli.Context) error {
 	return nil
 }
 
+var payInvoiceCommand = cli.Command{
+	Name:      "payinvoice",
+	Usage:     "pay an invoice over lightning",
+	ArgsUsage: "pay_req",
+	Flags: []cli.Flag{
+		cli.StringFlag{
+			Name:  "pay_req",
+			Usage: "a zbase32-check encoded payment request to fulfill",
```

Comment:
this should be changed to zpay32-encoded here and elsewhere.

Reply:
I'm also not sure you really need the `pay_req` flag, as this command takes only one argument. Instead you could add a `Description` field.

Reply:
@halseth I am happy to do that. But I added that to keep it consistent with other single argument commands. decodePayReqComamnd is an example.

Reply:
Ok, makes sense. Guess it doesn't hurt to have it as an option :)